modules/builds-secrets-restrictions.adoc

```diff
@@ -10,7 +10,12 @@ To use a secret, a pod needs to reference the secret. A secret can be used with
 * As files in a volume mounted on one or more of its containers.
 * By kubelet when pulling images for the pod.
 
-Volume type secrets write data into the container as a file using the volume mechanism. `imagePullSecrets` use service accounts for the automatic injection of the secret into all pods in a namespaces.
+Volume type secrets write data into the container as a file using the volume mechanism. `imagePullSecrets` use service accounts for the automatic injection of the secret into all pods in a namespaces. 
+
+[NOTE]
+====
+When creating secrets to store image pull information using the `imagePullSecrets` object, you cannot use the `{serviceaccount-name}-dockercfg` pattern. When this pattern is used, the `openshift-controller-manager` does not create a token or pull secret for that service account.
```

Comment:
See https://github.com/openshift/openshift-docs/pull/83900/files#r1810890086 
I prefer to `creating secrets for`
Other looks good to me, thanks